alerting-config-service-api/src/main/proto/org/hypertrace/alerting/config/service/v1/event_condition.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alerting.config.service.v1;
+
+import "org/hypertrace/alerting/config/service/v1/metric_anomaly_event_condition.proto";
+
+message NewEventCondition {
```

Comment:
One pattern that we've started adopting in other apis - pull the editable parts of a message out into their own shared message rather than repeating then. This makes it easier to work with in code for things like validation, and allows changes to the mutable part of the message to be made in one spot. Changes to the immutable part of the message 

```proto
message CreateMyMessageRequest {
  MyMessageData data = 1;
}

message CreateMyMessageResponse {
  MyMessage message = 1;
}

message UpdateMyMessageRequest {
  string id = 1;
  MyMessageData data = 2;
}

message UpdateMyMessageResponse {
  MyMessage message = 1;
}

message MyMessage {
  string id = 1;
  // Other non-editable fields
  MyMessageData data = 2;
}

message MyMessageData {
  oneof condition { // For example
    MetricAnomalyEventCondition metric_anomaly_event_condtion = 1;
  }
}
```

Reply:
That has just one problem coming up with a decent name for the shared message, I actually tried this while making the draft change itself but gave up 😅

Reply:
Yeah we've been using `Data` which I don't love. Maybe `MutableData` would be more descriptive, since that's how I phrased it when describing it in words...